### ISSUE TITLE
Question about a skin feature

### ISSUE THREAD

**Issue by TheGloriousArgonaut** [jaburns/NintendoSpy#66]
Is there any way to make a transparent skin for the overlay, by not using a chroma key?

I'd like to have the input display on a top window above my dolphin window so I can actually see my inputs in real time without the input window blocking too much screen space.

Is this possible?

I already tried making the default background Tron skin transparent in CS6 but then the transparent .png is rendered as pure white in the application.

Basically is there any way for the window to be opaque or transparent so I can put it on top of my actual dolphin window and see through it to the game? Obviously the windows window will still be seen but trying to make it as 'see through' as possible?

**Comment by jaburns** [comment]
It should be possible but would definitely require code changes. Unfortunately I'm not actively developing this project but PRs are welcome.
